src/main/java/com/julienvey/trello/domain/CustomFieldItem.java

```diff
@@ -0,0 +1,76 @@
+package com.julienvey.trello.domain;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class CustomFieldItem extends TrelloEntity {
```

Comment:
this is sent over the wire, let's mark it with json attributes ,like other POJOs